odl/tomo/geometry/parallel.py

```diff
@@ -713,31 +905,115 @@ def det_axes_init(self):
         return self.detector.axes
 
     def det_axes(self, angles):
-        """Return the detector axes tuple at ``angle``."""
-        return tuple(self.rotation_matrix(angles).dot(axis)
-                     for axis in self.det_axes_init)
+        """Return the detector axes tuple at ``angles``.
+
+        Parameters
+        ----------
+        angles : `array-like` or sequence
+            Euler angles in radians describing the rotation of the detector.
+            The length of the provided argument (along the first axis in
+            case of an array) must be equal to the number of Euler angles
+            in this geometry.
+
+        Returns
+        -------
+        axes : `numpy.ndarray`
+            Unit vector(s) along which the detector is aligned.
+            If ``angles`` is a single pair (or triplet) of Euler angles,
+            the returned array has shape ``(2, 3)``, otherwise
+            is ``(2,) + broadcast(*angles).shape + (3,)``.
```

Comment:
Not quite sure about this ordering. Opinion? Is it better to just add the extra dimensions to the left?

Reply:
I'd say that they should go to the left, more consistent with other stuff